### PR TITLE
Fix crash when connecting to a meeting with an attendee already in meeting with content share

### PR DESCRIPTION
**Pull request by rlebosse**
**Issue #:** 1153

**Description of changes:** Check that the content share stream if defined before getting its media tracks

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/contentsharecontroller/DefaultContentShareController.ts [hunk 1]
```diff
@@ -147,7 +147,7 @@ export default class DefaultContentShareController
           return;
         }
         const stream = this.mediaStreamBroker.mediaStream;
-        if (stream.getVideoTracks().length > 0) {
+        if (stream && stream.getVideoTracks().length > 0) {
```
Comment (richnew10) [inline src/contentsharecontroller/DefaultContentShareController.ts:150]: This can be more simply worded as

```
if (stream?.getVideoTracks().length) {
```

**Review by richnew10 — COMMENTED** [review]
From reading #1153 it looks like the confusion here is that the SDK is conflating "I have started content share and thus have a media stream" with "there is an attendee in the meeting with my attendee ID that's sharing content", and the latter is only an approximation of the former.

This fix might avoid one particular bug, but it's probably worth looking more deeply to consider whether there is a more precise phrasing (e.g., adding `|| !this.mediaStreamBroker.mediaStream` to the early return above) or a deeper check that needs to occur elsewhere.

File: src/contentsharecontroller/DefaultContentShareController.ts (hunk 1 above)
Comment (rlebosse) [reply src/contentsharecontroller/DefaultContentShareController.ts]: Totally, I have made the modification. Thanks for your feedback @richnew10 !

**Comment by ltrung** [comment]
@rlebosse Thanks for fixing this :). I have created a new PR #1265 with your changes and rebased it and merged it in.

**Pull request closed without merging**